akka-typed/src/main/scala/akka/typed/patterns/Restarter.scala

```diff
@@ -7,6 +7,52 @@ package patterns
 import scala.reflect.ClassTag
```

Comment:
we have to think about if we should use a `patterns` package, and if so do we need `scaladsl.patterns` and `javadsl.patterns`